stdlib/builtins.pyi

```diff
@@ -156,6 +156,7 @@ class type(object):
     def __subclasscheck__(self, subclass: type) -> bool: ...
     @classmethod
     def __prepare__(metacls, __name: str, __bases: Tuple[type, ...], **kwds: Any) -> Mapping[str, Any]: ...
+    def __or__(self, t: type) -> type: ...
```

Comment:
You could wrap this inside `if sys.version_info >= (3, 10)`

Reply:
Yes, I forgot about it. I'll do it and create new one

Reply:
You can also change a pull request by pushing more commits to your branch.

Reply:
Ok, next time I will do this way